nat/nat.go

```diff
@@ -112,12 +113,15 @@ func (dev *Mapper) keepPortMapping(mappedPort chan<- uint16, protocol string,
 			for {
 				select {
 				case <-updateTimer.C:
-					if err := dev.r.MapPort(protocol, intPort, extPort, desc, mapTimeout); err != nil {
-						dev.log.Error("Renewing port mapping from external port %d to internal port %d failed with %s",
-							intPort, extPort, err)
-					} else {
-						dev.log.Debug("Renewed port mapping from external port %d to internal port %d.",
-							intPort, extPort)
+					for retryCnt := 0; retryCnt < maxRefreshRetries; retryCnt++ {
```

Comment:
should we sleep for a second between refresh attempts?
Not sure if this would actually make the subsequent attempt more likely to succeed

Reply:
The sleep was added.